src/Particular.PlatformSample.ServiceControl/Particular.PlatformSample.ServiceControl.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net461</TargetFramework>
+    <IncludeBuildOutput>False</IncludeBuildOutput>
+    <SuppressDependenciesWhenPacking>True</SuppressDependenciesWhenPacking>
+    <Description>Particular ServiceControl binaries for use by Particular.PlatformSample. Not intended for use outside of Particular.PlatformSample.</Description>
+    <PackageProjectUrl>https://docs.particular.net/servicecontrol/</PackageProjectUrl>
+  </PropertyGroup>
```

Comment:
You also need to add `<NoWarn>$(NoWarn);NU5118;NU5100</NoWarn>` here.

Reply:
Done